microsoft-edge/devtools-guide-chromium/memory-problems/dom-leaks.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Debug DOM memory leaks
+description: Use the Detached Elements tool in the Microsoft Edge DevTools to find and fix DOM memory leaks.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 11/30/2021
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools, detached elements, document object model, dom, memory, leak, node
+---
+
+# Debug DOM memory leaks with the Detached Elements tool
+
+A memory leak can occur in your application when an element is no longer attached to the Document Object Model (DOM) tree, but is still referenced by some JavaScript running on the page. These elements are called *detached elements*. For the browser to garbage collect (GC) the detached element, the element must not be referenced from either the DOM tree or JavaScript code.
+
+This article describes how to use the **Detached Elements** tool in Microsoft Edge DevTools to find detached elements that the browser cannot GC in a demo application and identify the JavaScript that is still referencing the detached element. By changing your JavaScript to release the element, you reduce the number of detached elements on your page.
+
+For more information about heap snapshots and detached elements, read [Discover detached DOM tree memory leaks with Heap Snapshots](index.md#discover-detached-dom-tree-memory-leaks-with-heap-snapshots).
+
+
+<!-- ====================================================================== -->
+## Open the Detached Elements tool
+
+The **Detached Elements** tool is available by default in Microsoft Edge 97 and later. Check your version of Microsoft Edge by going to `edge://version`. If your version is less than 97, read [Turning an experiment on or off](../experimental-features/index.md#turning-an-experiment-on-or-off) for instructions about turning on the **Detached Elements** experiment.
+
+To open the **Detached Elements** tool:
+
+1. Open the [demo application](https://microsoftedge.github.io/Demos/detached-elements/).
+
+1. [Open DevTools](../open/index.md).
+
+1. Click the **+** button in the toolbar.
+
+1. Click **Detached Elements**.
+
+:::image type="content" source="./images/open-detached-elements.msft.png" alt-text="Open the Detached Elements tool." lightbox="./images/open-detached-elements.msft.png":::
+
+
+<!-- ====================================================================== -->
+## Get detached elements
+
+The **Get Detached Elements** (![The Get Detached Elements icon.](images/get-detached-elements-icon-light-mode.png)) button in the **Detached Elements** tool finds and displays all of the detached elements on a web page.
+
+To find detached elements:
+
+1. In the demo application, click the **Room 1** button.
+
+1. In the demo application, click the **Simulate high traffic** button.
+
```

Comment:
@zoherghadyali - It would be good to add a screenshot showing content in the webpage on left & DevTools on right.